templates/org/team/teams.tmpl

```diff
@@ -16,6 +16,7 @@
 					<div class="ui top attached header">
 						<a class="text black" href="{{$.OrgLink}}/teams/{{.LowerName | PathEscape}}"><strong>{{.Name}}</strong></a>
 						<div class="ui right">
+							<a class="btn ui primary small button" href="{{$.OrgLink}}/teams/{{.LowerName | PathEscape}}">{{ctx.Locale.Tr "org.teams.view"}}</a>
```

Comment:
There is an exactly same link above: `<a class="text black" href="{{$.OrgLink}}/teams/{{.LowerName | PathEscape}}">`


Why duplicate?

Reply:
@wxiaoguang In the linked issue (#22054), multiple users (including myself) didn't realize that the title of the group is also a link. It's only visible on hover. I am not opposed to removing the link. The purpose of this PR is to make it more obvious. Will defer to you, @lunny and @Sonic853 on whether or not we should keep the link in the title.